### PR TITLE
ODA-1137 there should be no changelogs exported from a new draft

### PR DESCRIPTION
**Pull request by rickreesrr**

### CODE REVIEW

File: ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/publishing/changelog/AbstractChangeLogGenerator.groovy [hunk 1]
```diff
@@ -271,6 +271,18 @@ abstract class AbstractChangeLogGenerator {
...
+            ne 'undone', true
+            isNull 'parentId'
+            gt 'dateCreated', startDate
+
+            if (types) {
+                inList 'type', types.toList()
+            }
+        }.items
+    }
+
```
Comment (musketyr) [inline ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/publishing/changelog/AbstractChangeLogGenerator.groovy]: do you keep in mind the correct order of the columns to fit to existing indexes?

Comment (musketyr) [inline ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/publishing/changelog/AbstractChangeLogGenerator.groovy]: nvm, it looks like there is no matching index. we'll if we need to add more

**Comment by rickreesrr** [comment]
I assumed it was hitting the index as it was still very fast when I tested it.

**Comment by musketyr** [comment]
ok, cool :-)

**Pull request merged**